packages/external-db-config/lib/readers/aws_config_reader.js

```diff
@@ -23,77 +21,85 @@ class AwsConfigReader {
     return JSON.parse(data.SecretString)
   }
 
+  async getExternalAndLocalEnvs() { 
+    const externalConfig = await this.readExternalConfig().catch(emptyConfig)
+    const { host, username, password, DB, SECRET_KEY, HOST, PASSWORD, USER } = { ...process.env, ...externalConfig }
+    const config = {  host: host || HOST, username: username || USER, password: password || PASSWORD, DB, SECRET_KEY }
+    return { config, secretMangerError: externalConfig.secretMangerError }
+  }
+
   async validate() {
-    try {
-      const cfg = await this.readExternalConfig()
-      return { missingRequiredSecretsKeys: checkRequiredKeys(cfg, ['host', 'username', 'password', 'DB', 'SECRET_KEY']) }
-    } catch (err) {
-      return { configReadError: err.message, missingRequiredSecretsKeys: [] }
-    }
+      const { config, secretMangerError } = await this.getExternalAndLocalEnvs()
+      return { missingRequiredSecretsKeys: checkRequiredKeys(config, ['host', 'username', 'password', 'DB', 'SECRET_KEY']), secretMangerError }
```

Comment:
If we are already editing the file, maybe it will be a good idea to create a new variable for the `RequiredKeys` array.